testing/MLDB-2181_null_feature_model_test.py

```diff
@@ -0,0 +1,164 @@
+                        "verbosity": 3,
+                        "weak_learner": {
+                            "type": "boosting",
+                            "verbosity": 3,
+                            "weak_learner": {
+                                "type": "decision_tree",
+                                "verbosity": 0,
+                                "max_depth": 5,
+                                "random_feature_propn": .1,
+                            },
+                            "min_iter": 5,
+                            "max_iter": 30
+                        },
+                        "num_bags": 5
+                    }
+                },
+                "functionName": 'score_it',
+                'modelFileUrl': 'file://' + cls.model_file.name
+            }
+        })
+
+    def test_string_over_null(self):
+        ds = mldb.create_dataset({
+            'id': 'test_string_over_null_ds',
+            'type': 'sparse.mutable'
+        })
+        for idx in xrange(10):
+            ds.record_row('row{}'.format(idx), [
+                ['line', idx, 0],
+                ['label', 0, 0],
+                ['feature', random.random() + 0.6, 0],
+                ['noise', random.random(), 0],
+                ['issue', 'STRING', 0]
+            ])
+        ds.commit()
+
+        mldb.post('/v1/procedures', {
+            'type': 'classifier.test',
+            'params' : {
+                'mode': 'boolean',
+                'testingData':
+                    'SELECT score_it({features: {feature, noise, issue}})[score] AS score, ' \
+                    'label FROM test_string_over_null_ds'
+            }
+        })
+
+    def test_num_over_null(self):
+        ds = mldb.create_dataset({
+            'id': 'test_num_over_null_ds',
+            'type': 'sparse.mutable'
+        })
+        for idx in xrange(10):
+            ds.record_row('row{}'.format(idx), [
+                ['line', idx, 0],
+                ['label', 0, 0],
+                ['feature', random.random() + 0.6, 0],
+                ['noise', random.random(), 0],
+                ['issue', random.random() * 10, 0]
+            ])
+        ds.commit()
+
+        mldb.post('/v1/procedures', {
+            'type': 'classifier.test',
+            'params' : {
+                'mode': 'boolean',
+                'testingData':
+                    'SELECT score_it({features: {feature, noise, issue}})[score] AS score, ' \
+                    'label FROM test_num_over_null_ds'
+            }
+        })
+
+    def test_c_over_a_or_b(self):
+        ds = mldb.create_dataset({
+            'id': 'test_c_over_a_or_b_ds',
+            'type': 'sparse.mutable'
+        })
+        for idx in xrange(10):
+            ds.record_row('row{}'.format(idx), [
+                ['line', idx, 0],
+                ['label', 0, 0],
+                ['feature', random.random() + 0.6, 0],
+                ['noise', random.random(), 0],
+                ['a_or_b', 'c', 0]
```

Comment:
This wasn't the same problem, right?  Here it is a new value but the type does not change.